### PR TITLE
Add Data Control Plane link to doc index [ci-skip]

### PR DESCRIPTION
**Pull request by travis79**
Adding the new data-control-plane docs section to the SUMMARY.md index

### CODE REVIEW

File: docs/user/SUMMARY.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
     - [Validating metrics](user/metrics/validation-checklist.md)
     - [Error reporting](user/metrics/error-reporting.md)
     - [Metrics collected by the Glean SDKs](user/collected-metrics/metrics.md)
-    - [Enable and Disable metrics using Nimbus](user/metrics/metrics-remote-settings.md)
+    - [Data Control Plane (Remote Metric Configuration)](user/metrics/data-control-plane/index.md)
```
Comment (badboy) [inline docs/user/SUMMARY.md:20]: you will need to link all the subpages as well.

Comment (travis79) [reply docs/user/SUMMARY.md:20]: On a side-note...  does [ci-skip] not skip building on Taskcluster?

Comment (badboy) [reply docs/user/SUMMARY.md:20]: only in the PR title :/

Comment (travis79) [reply docs/user/SUMMARY.md:20]: But...  it is in the PR title?

Comment (badboy) [reply docs/user/SUMMARY.md:20]: oh, it must be `[ci skip]` (no dash, https://mozilla.github.io/glean/dev/ci.html#skipping-ci-completely)

**Pull request merged**